src/DynamicForm/RepeatableForm.js

```diff
@@ -1,49 +1,43 @@
-import React, { useState, useRef } from 'react'
+import React, { useState, useRef, useCallback } from 'react'
 
 import { NodeForm } from './NodeForm'
 import util from '../util'
 
 let keyIndex = 0
-export const ReapeatableForm = ({ schema, onChange = () => {} }) => {
+const _ReapeatableForm = ({ schema, atKey = null, onChange = () => {} }) => {
```

Comment:
for onChange, we could do something like `() => void` which looks much cleaner 
PS: not sure if works here

Reply:
Ohh! I did not know that.
But I am going to use [`prop-types`](https://www.npmjs.com/package/prop-types), So that change is not required.
Thanks.